Makefile

```diff
@@ -252,3 +257,7 @@ misspell:
 .PHONE: kind-e2e-test
 kind-e2e-test:
 	test/e2e/run.sh
+
+.PHONE: run-ingress-controller
```

Comment:
What's the difference between `PHONE` and `PHONY`?

Reply:
fixed.